slide-deck.js

```diff
@@ -455,9 +455,12 @@ class slideDeck extends HTMLElement {
     this.goTo(this.slideFromStore());
   }
 
+  // Detect Ctrl / Cmd modifiers in a platform-agnostic way
+  cmdOrCtrl = (event) => event.ctrlKey || event.metaKey;
```

Comment:
I think this should be a private method (the other PR is making most of the others private, so this would match that, eventually).